README.md

```diff
@@ -37,6 +37,8 @@ It reads your :key: ACF PRO key from the **environment** or a **.env file**.
 ```
 Replace `"version": "*.*.*(.*)"` with your desired version.
 
+Replace `"type": "wordpress-plugin"` with `"type": "library"` if you would like to have ACF PRO installed in the `./vendor` directory instead of `./wp-content/plugins/`. This may be desireable if for example, you are including ACF PRO in a Wordpress theme.
```

Comment:
Would you mind removing the `/` after `./wp-content/plugins/` for consistency with `./vendor` and replacing Wordpress with Word**P**ress to make [Matt](https://de.wikipedia.org/wiki/Matthew_Mullenweg) happy? ☺️